canonical/ubuntu.com
# WD-8838 -  add instagram to the list of social media

## Done

Add instagram to the list of social media in the footer

## QA
- [demo link](https://ubuntu-com-13598.demos.haus/)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the logo and link are correct for instagram

## Issue / Card
[WD-8838](https://warthogs.atlassian.net/browse/WD-8838)

Fixes #13572

## Screenshots

[WD-8838]: https://warthogs.atlassian.net/browse/WD-8838?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
